components/HelpComponents/HelpGuide.tsx

```diff
@@ -6,12 +6,10 @@ import styles from "../../styles/HelpGuide.module.css";
 import { styled } from '@mui/material/styles';
 import IconButton from '@mui/material/IconButton';
 import CloseIcon from '@mui/icons-material/Close';
```

Comment:
Are we converting all of the import statements from ' ' to " " ? It would be good to have it consistent either way. Let me know and I can do it if you want :).

Reply:
Yea, I can't review at this state

Reply:
sure I'll change it to ' '

Reply:
I'll fix the '' issue in the prettify codebase branch